lib/new_relic/ecto/telemetry.ex

```diff
@@ -0,0 +1,83 @@
+defmodule NewRelic.Ecto.Telemetry do
+  use GenServer
+
+  @handler_id :new_relic_ecto
+
+  def start_link(metrics: metrics) do
+    GenServer.start_link(__MODULE__, metrics: metrics)
+  end
+
+  def init(metrics: metrics) do
+    Process.flag(:trap_exit, true)
+
+    Enum.each(metrics, fn metric ->
+      :telemetry.attach(
```

Comment:
I did have that before but I was attempting to get the `metric` in the config so that it could be used when reporting (perhaps to get the `metric.name` and use it as an attribute or something

Are you saying that you can only call `attach` once per `handler_id`?

I could pass all the metrics into `config` and pull the right one out later

Reply:
Yes, the handler ID is unique :)